Products/PleiadesEntity/content/Named.py

```diff
@@ -119,6 +120,30 @@ def getLocations(self):
                  or checkPermission('Pleiades: View link to draft', o))
         ]
 
+    security.declareProtected(permissions.View, 'getLocations')
+    def getSubConnections(self):
+        """
+        """
+        checkPermission = getSecurityManager().checkPermission
+        return [
+            o for o in self.values()
+            if interfaces.IConnection.providedBy(o)
+            and (checkPermission(permissions.View, o)
+                 or checkPermission('Pleiades: View link to draft', o))
+        ]
+
+    security.declareProtected(permissions.View, 'getLocations')
+    def getReverseConnections(self):
+        """
+        """
+        checkPermission = getSecurityManager().checkPermission
+        return [
+            o for o in self.getBRefs('connection')
+            if interfaces.IConnection.providedBy(o)
+            and (checkPermission(permissions.View, o)
+                 or checkPermission('Pleiades: View link to draft', o))
+        ]
```

Comment:
I believe Features are no longer used (though there are some unpublished ones remaining in an archive) and the intent is to remove them at some point, right @paregorios?